doc/decisions/backend_api_documentation.md

```diff
@@ -0,0 +1,52 @@
+# Backend API documentation
+
+## Problem
+
+We need a way to document the API in the backend. Otherwise we would have to read the Rust Code every time we want to develop a new part of the frontend.
+
+## Constraints
+
+## Assumptions
+
+## Solutions
+
+### Utoipa
+
+You can think of [Utoipa](https://github.com/juhaku/utoipa) as the Rust equivalent to Swagger/OpenApi in Java.
+
+It is a Rust crate which allows you to annotate your code (structs and endpoints) and automatically generate OpenApi documentation from that. You can also generate a UI endpoint just like with Java Spring.
+
+Compared to swagger in Java there is a bit more you have to configure to get it running.
+
+Once you get it running you get endpoints with schemas and examples representing the API. You can also execute example requests directly from the UI to see how the backend behaves.
+
+### API Blueprint
+
+Information from <https://testfully.io/blog/api-blueprint/>.
+
+Before starting on a new endpoint we would write a markdown document specifying in detail what the endpoint should return and in what format. There is also a [lot of tooling](https://apiblueprint.org/tools.html) for API Blueprint.
+
```

Comment:
Please write also about other alternatives you found. Some investigations should be done.

Reply:
Do you mean alternatives in the sense of crates/mechanisms similar to Utoipa or API blueprint? Or rather investigate API blueprint in more detail?

Reply:
I meant other crates but you already said it in the meeting: it seems like there is not really something else available.